packages/ptokens-assets-eosio/test/ptokens-eosio-asset.test.ts

```diff
@@ -342,6 +389,66 @@ describe('EOSIO asset', () => {
       }
     })
 
+    test('Should call transact with custom actions', async () => {
```

Comment:
well done!

Reply:
I would just separate from the following test specifing that this is for native asset and the other one no.